src/main/java/seedu/address/ui/CardCard.java

```diff
@@ -45,7 +45,10 @@ public CardCard(Card card, int displayedIndex, List<Tag> tagList) {
         front.setText(card.getFront());
         tagList.forEach(tag -> tags.getChildren().add(new Label(tag.getName().toString())));
         if (card.getType().equals(McqCard.TYPE)) {
-            card.getOptions().forEach(option -> options.getChildren().add(new Label(option)));
+            for (int i = 0; i < card.getOptions().size(); i++) {
```

Comment:
any reason you changed this?
could be just:

`card.getOptions().forEach(option -> options.getChildren().add(new Label((i + 1) + ") " + option))); `

Reply:
But there's no index i if I do that?

Reply:
oh yeah oops nvm